### DISCUSSION TITLE
Order entity has no `tags` field

### DISCUSSION THREAD

**Discussion by duailibe** [Shopify/storefront-api-feedback#146] [Ideas & Requests]
The [Order](https://shopify.dev/api/storefront/unstable/objects/Order) object in the Storefront API has no `tags` field, which are available in the Liquid object.

Was that intentional? Is it possible to request it to be added?

**Comment by johncraigcole** [comment]
Hey @duailibe I've added an enhancement request for this on the Shopify end but have not timeline on when or if this enhancement request will be implemented. That being said, I was curious about the use case how the order tags would be used if you're comfortable sharing that.

**Reply by duailibe** [reply]
@johncraigcole Thanks for the response! We currently have some a few manual payment methods and used to store the which payment method was used on tags. We use that info to infer some state on that order to display to the user.

Now that we're moving to a headless store, that info was lost since we can't read tags, nor the payment gateway chosen.

We've actually changed that now. We're using metafields for that info. It took a while to migrate everything, but eventually unblocked us.

**Reply by johncraigcole** [reply]
Thanks for all that context. I was about to suggest metafields as a workaround and happy to hear you were unblocked. I've got the tags enhancement on the record so hoping if resourcing permits it's something we can add in the future.
